src/components/Fingerprint.vue

```diff
@@ -166,5 +168,21 @@ export default class Fingerprint extends ( mixins( Messages ) as VueConstructor<
 			@include fontSize( 23px );
 		}
 	}
+	.wikibase-termbox-fingerprint-list & {
```

Comment:
This is a class (only) existing in a parent component. No good.
If this is to negate the `--primaryLanguage` modifier we could
* use a second modifier and use that whenever we are not showing primary language content
* define the default, non-primary language CSS first, and explicitly override what is different for the primary language

I'm leaning towards the latter but it sure _could_ be verbose...

Reply:
Using `:not` seems fine!